CGPs/0017.md

```diff
@@ -84,14 +84,16 @@ In response to the feedback from the community, we have decided to narrow down o
 
 **Other:** In addition to the above three areas, we will be open to considering other great teams/projects that might be beneficial to Celo’s mission/ecosystem.
 
-#### 5.5 Funding Projects
+#### 5.5 Funding Projects*
 We will target projects that we believe will be able to deliver measurable key results in exchange for 20k-33k CELO. We will target funding 30 projects which would give an average funding amount of 22,180. This parameter set would enable us to for example, fund 10 small projects (<5,000 CELO), 15 projects of medium size (~22,180 CELO), and 2 large projects (>40,000 CELO). These targets are not set in stone.
 
+*assuming a USD/CELO price of 1.77. Note: The price of CELO is always changing based on market conditions. The price was calculated in Dec 2020 as an average of 90 days.
```

Comment:
Given this PR was submitted today, can we have a new 90 day average price of CELO for January 2020?

Reply:
I can update this section but will have to change the other sections and numbers as well for consistency then. WE have been updating the number since Nov. IT's just a moving target and the number will keep changing